clusterloader2/pkg/prometheus/manifests/0prometheus-operator-deployment.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         - --kubelet-service=kube-system/kubelet
         {{end}}
         - --prometheus-config-reloader=gcr.io/k8s-testimages/quay.io/prometheus-operator/prometheus-config-reloader:v0.46.0
+        - --secret-field-selector=prometheus-operator=true
```

Comment:
"prometheus-operator=true"
looks more like label-selector to me, then field-selector

I guess we can only filter on:
- metadata.name
- metadata.namespace
- type
https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/secret/strategy.go#L118

Reply:
Frankly speaking I thought that it's labelSelector :D I haven't read the name of the flag carefully. Changed to "type"